doc/report.Rmd

```diff
@@ -53,14 +55,14 @@ Each point in which a conditional statement is evaluated and the data could pote
 Splits are typically evaluated by Gini impurity or entropy.
 
 $$
-Gini\ Impurity\ =\ I_G(t)\  = 1 - \sum_{i=1}^{C}p(i|t)^2
+\text{Gini Impurity} =\ I_G(t)\  = 1 - \sum_{i=1}^{C}p(i|t)^2
 $$
 
 $$
-Entropy\ =\ I_H(t)\ = -\sum_{i=1}^{C}p(i|t)log_2p(i|t)
+\text{Entropy} =\ I_H(t)\ = -\sum_{i=1}^{C}p(i|t)log_2p(i|t)
 $$
```

Comment:
Use `\text{}` around any full words in your equations - otherwise LaTeX typesets each letter as though it is a variable, and the spacing gets wacky.  I've fixed it here, but you need to double check your other equations and make the same fix.

Reply:
Also add a backslash in front of log, e.g. `\log_2{p(i|t)}` to get it typeset a bit nicer too.